API.md

```diff
@@ -0,0 +1,388 @@
+  ```
+
+  Prints 'Hello World!', highlighted (i.e., white on black with the default colors), centered & padded on both sides, on the fifth line starting from the bottom.
+
+* ```sh
+  fbink -pmM -y -8 "Hello World!"
+  ```
+
+  Prints 'Hello World!', centered & padded on both sides, eight lines above the center of the screen.
+
+## Options
+
+### Options affecting the message's position on screen
+
+* `-x`, `--col` `NUM`
+
+  Begin printing `STRING` @ column `NUM` (Default: 0).
+
+  You might consider beginning at column 1 instead of 0, as column 0 (the leftmost one) may sometimes be slightly obscured by a bezel.
+
+  Use a negative value to count back from the right edge of the screen.
+
+* `-y`, `--row` `NUM`
+
+  Begin printing `STRING` @ row `NUM` (Default: 0).
+
+  You might consider beginning at row 1 instead of 0, as row 0 (the topmost one) may sometimes be slightly obscured by a bezel, especially on Kobos.
+
+  Use a negative value to count back from the bottom of the screen.
+
+* `-`, `--hoffset` `NUM`
+
+  Adjust final text position by `NUM` pixels on the horizontal axis (Default: 0).
+
+  Honors negative values, and will let you push stuff off-screen, often without warning.
+
+* `-Y`, `--voffset` `NUM`
+
+  Adjust final text position by `NUM` pixels on the vertical axis (Default: 0).
+
+  Honors negative values, and will let you push stuff off-screen, often without warning.
+
+* `-m`, `--centered`
+
+  Dynamically override col to print `STRING` at the center of the screen.
+
+  Special care is taken to avoid the very edges of the screen, to ensure the complete legibility of the message.
+
+* `-M`, `--halfway`
+
+  Dynamically adjust row to print `STRING` in the middle of the screen.
+
+  The value specified in row then becomes an offset, starting from the middle of the screen.
+
+* `-p`, `--padded`
+
+  Left pad `STRING` with blank spaces.
+
+  Most useful when combined with --centered to ensure a line will be completely filled, while still centering `STRING`, i.e., padding it on both sides.
+
+* `-r`, `--rpadded`
+
+  Right pad `STRING` with blank spaces.
+
+### Options affecting the message's appearance
+
+* `-h`, `--invert`
+
+  Print `STRING` in `BACKGROUND_COLOR` over `FOREGROUND_COLOR` instead of the reverse. (See `-B` and `-C`)
+
+* `-f`, `--flash`
+
+  Ask the eInk driver to do a black flash when refreshing the area of the screen where `STRING` will be printed.
+
+  Note if compiled with `FBINK_FOR_KINDLE`: on legacy einkfb devices, this may not always be honored by the hardware.
+
+* `-c`, `--clear`
+
+  Clear the full screen before printing.
+
+  Honors `-B`, `--background`; `-h`, `--invert`; `-H`, `--nightmode`; `-W`, `--waveform`; `-D`, `--dither`; `-b`, `--norefresh`; `-w`, `--wait`
+
+  Can be specified on its own, without any `STRING`.
+
+  NOTE: If your intent is to simply clear the screen and *nothing else*, use `-k`, `--cls` instead!
+
+* `-W`, `--waveform` `MODE` (not available if compiled with `FBINK_FOR_LINUX`)
```

Comment:
In the spirit of my earlier `FBINK_FOR` comment, that ones means "not an e-Ink device".

Reply:
Fixed in 7e4ab88586c01abc352eac313b266afb759e56e1